src/sudoku.py

```diff
@@ -169,3 +170,17 @@ def build_grid():
                     break
             i += 1
         return sudokus[9].get_grid()
+
+    @classmethod
+    def get_task(self):
+        task = Sudoku.build_grid()
+        for index in random.sample(range(81), 81):
+            i = index // 9
+            j = index % 9
+            current_cell_copy = task[i][j]
```

Comment:
Имена переменных следует адаптировать к изменениям логики.
```suggestion
            true_cell_value = task[i][j]
```